src/slices/HeadSlice.tsx

```diff
@@ -48,9 +48,14 @@ const SpotDescription = () => {
   </VStack>
 }
 
+const AgendaButton = () => {
+   return <Button size='lg' variant='solid' colorScheme='purple'>Zobacz agendę</Button>
```

Comment:
nitpick: wrong color - should be #6F2DBD.
nitpick: wrong text - should be uppercased.
Could You extract Button into another reusable component?

Reply:
Good job!

Reply:
Changed  the colour and the text and created some sort of component. I don't know if thios is the desired way to define properties (as an interface) but I found this kind of explanation here - https://stackoverflow.com/questions/65063749/how-to-set-default-value-of-props-if-not-passed-to-functional-component-in-nextj
Let me know what I can improve and if there's a dedicated way to create components along with their parameters and their default values 😉

Reply:
It is a good way to define a props. Interface is a specific case and more optimal than `type` keyword, but it doesn't matter in such a small project. Props is just an object, so it should be called `props` (lowercase). I think the most popular way to define default property in react component is to make destructurization at function definition ([example](https://github.com/asi-pwr/17.sesja.linuksowa-website/pull/5/files#diff-97d978b367f7d401c68d4a22d73bf6e38f37b6811654ce43fc6096e8890a7773R10), but your way is also a good one :)